### PR TITLE
Stakeholder loading from Tygron

### PR DESCRIPTION
**Pull request by PauluzzNL**
Loading of stakeholders from Tygron. This required some changes in the Connections as well to work with the tokens. Concerns #73

### CODE REVIEW

**Comment by HarmGriffioen** [comment]
:+1:

File: tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/Connection.java [hunk 1]
```diff
@@ -35,4 +35,6 @@ public JSONObject callPostEventObject(String eventName, JSONArray parameters) {
   public JSONArray callPostEventArray(String eventName, JSONArray parameters) {
     return new JSONArray(callPostEvent(eventName, parameters));
   }
+  
+  public abstract void setServerToken(String serverToken) ;
```
Comment (devbridie) [inline tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/Connection.java]: remove space before ;

Comment (devbridie) [inline tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/Connection.java]: Also why is this abstract?

**Comment by JoeHarrison** [comment]
:+1:

**Comment by devbridie** [comment]
:+1:

**Pull request merged**